db/migrations/2025_10_09_000000_add_email_food_to_user_settings.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Engelsystem\Migrations;
+
+use Engelsystem\Database\Migration\Migration;
+use Illuminate\Database\Connection;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Schema\Builder as SchemaBuilder;
+
+class AddEmailFoodToUserSettings extends Migration
+{
+    protected Connection $db;
+
+    public function __construct(SchemaBuilder $schema)
+    {
+        parent::__construct($schema);
+        $this->db = $this->schema->getConnection();
+    }
+
+    /**
+     * Run the migration
+     */
+    public function up(): void
+    {
+        $this->schema->table('users_settings', function (Blueprint $table): void {
+            $table->boolean('email_food')->after('email_goodie')->default(true);
+        });
+        $this->schema->table('users_state', function (Blueprint $table): void {
+            $table->json('meal_vouchers')->after('got_voucher')->nullable()->default(null);
```

Comment:
Meal or food? Imho it should be the same wording.
As the days are saved here, a wording change would be great too (not prevent confusion / make it clear that we are NOT saving the voucher codes)